lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -621,6 +668,15 @@ instance ToText StakePoolLog where
             ]
         MsgRollingBackTo point ->
             "Rolling back to " <> pretty point
+        MsgStakePoolGarbageCollection info bkt -> mconcat
+            [ "Performing garbage collection of retired stake pools. "
+            , "Currently in epoch "
+            , toText (poolGarbageCollectionCurrentEpoch info)
+            , ". Removing all pools that retired in or before epoch "
+            , toText (poolGarbageCollectionRemovalEpoch info)
+            , ". "
+            , toText bkt
+            ]
```

Comment:
See if this can be made simpler through use of [`Fmt`](http://hackage.haskell.org/package/fmt-0.6.1.2/docs/Fmt.html).